test/util_test.js

```diff
@@ -34,7 +34,7 @@ describe('newChartDefinition', () => {
   });
 });
 
-describe('isToUserDate', () => {
```

Comment:
The fix to this typo should probably be done regardless.

Reply:
True. I'll fix that today.